modules/standard/CommDiagnostics.chpl

```diff
@@ -21,7 +21,7 @@
   This module provides support for reporting and counting
   communication operations between network-connected locales.  The
   operations include various kinds of remote reads (GETs), remote
-  writes (PUTs), and remote forks (which are actually more like remote
+  writes (PUTs), and remote execute_ons (which are actually more like remote
```

Comment:
I appreciate the simplicity of just doing a straight string replace, but really this should be rephrased, since the parenthetical was mostly there to explain/clarify what "forks" meant. You probably could get away with "..., and remote executions", with no parenthetical.